atomate/vasp/fireworks/core.py

```diff
@@ -445,10 +446,9 @@ def __init__(self, structure, magmom, name="spin-orbit coupling",
 
 class TransmuterFW(Firework):
```

Comment:
I'm confused by these changes to TransmuterFW. In particular:

* the parent_structure argument is not documented, so I'm not sure what this is. The name of the parameter would make it seem like a Structure object, but it seems to be a bool that adds metadata. Why is not called ``add_metadata`` or something?
* I think I'm on board with the removal of the ``parents`` parameter. From what I can tell, ``**kwargs`` should handle that already. Is that correct? If so, I would suggest not changing it in this PR but submitting a separate PR that just cleans up all the extraneous ``parents`` parameters in all the various FWS (there are many that contain this)